src/NotificationStack.vala

```diff
@@ -22,7 +22,7 @@ public class Gala.NotificationStack : Object {
 
     // we need to keep a small offset to the top, because we clip the container to
     // its allocations and the close button would be off for the first notification
-    private const int TOP_OFFSET = 2;
```

Comment:
Since `TOP_OFFSET` is no longer a constant we should name it lowercase and move it away from other constants.

Reply:
I agree. I was thinking if I should use `TOP_OFFSET` as a `const` again, and use a second variable to control the offset. Because now I have the magic number 2 in two different places.

Reply:
Makes sense. That would be even better